config/util/config.go

```diff
@@ -14,6 +14,8 @@
 package util
 
 import (
+	"syscall"
```

Comment:
In the commit message for this commit, I'm not really sure what "helps to copy the file/dir mode" means.  Let's be sure to explain exactly what we're doing and why.